sample.py

```diff
@@ -0,0 +1,20 @@
+
+
+
+import sys
+import os
+def fib():
+    x = 0
```

Comment:
__[Flake8]__

local variable 'x' is assigned to but never used


<!-- Comment by [SideCI](https://sideci.com) -->